assignment2.txt

```diff
@@ -0,0 +1,29 @@
+Emma Ladouceur homework2 questions
+
+Question 1
+	Absolutely! Its quite common to mix two operators in equations. For example for the quadratic equation divition multiplication and square roots are all used. The important thing is remebering order of operations and parenthesis
+
+
+Question 2
+	Systax errors involve forgetting a ; or making a mistake in the coding of a program whereas logical errors are issues with the program and how it goes about completing its task.
+
+
+Question 3
+	True && x< 25 && x!=12
+	True && False && True
+	False && True
+	False (Which i believe in c is 0?)
+
+Question 4
+	I believe it is valid because  the entire statement is true. I'm not sure you could use it like that in code however. While both sides of the statement are true, I believe it should be written as 0<x && x<15 which means true && true --> so the answer is true.
```

Comment:
Right! The comparison operators are exclusively binary, and they cannot be used in sequence. To be valid, the expression should be written as some variation of 0 < x && x < 15.